apps/i18n/common/en_us.json

```diff
@@ -2887,6 +2887,7 @@
   "submitProjectGallery_header": "Submit to be featured",
   "submitProjectGallery_submitDescribeProject": "You can submit your project to be considered for our Featured Projects gallery. If you're interested, please describe the project: ",
   "submitProjectGallery_thanksForSubmission": "Thanks for your submission.",
+  "submitProjectGallery_tryAgain": "We were unable to submit your project. Please try again in one minute.",
```

Comment:
"one minute" seems oddly specific (as opposed to "Please try again"). Is there a reason for specifying a time?

Reply:
Please see thread at https://codedotorg.slack.com/archives/C04TH8400AU/p1730755091667999?thread_ts=1730745582.023389&cid=C04TH8400AU

Reply:
updated string looks good!!